### ISSUE TITLE
Toolbar is not responsive

### ISSUE THREAD

**Issue by alarstyle** [sitecake/sitecake#58]
Sometimes I need to edit page content fast and I'm using my phone for that. And the toolbar is not looking good on mobile right now, it's not easy to use.

**Comment by adaniello** [comment]
+1

**Comment by fallenartist** [comment]
+10 and it's not only mobile, also narrower windows on desktop. Exposed CSS to change the appearance of the toolbar would be nice.

**Comment by nikolamircic** [comment]
Agreed. We are pulling toolbar CSS out so it's easily accessible and skinnable. If you have some good examples how it's done in other projects -- speak now or forever hold your peace.

**Comment by fallenartist** [comment]
I dunno but maybe https://medium.com will inspire some good solutions? Looking forward to the update!

**Comment by fallenartist** [comment]
FontShop's [type tryout module](https://www.fontshop.com/tryout?referer=/families/ff-netto) also looks/works well!
